src/error/syntaxError.js

```diff
@@ -11,6 +11,29 @@
 import { getLocation } from '../language/location';
 import type { Source } from '../language/source';
 import { GraphQLError } from './GraphQLError';
+import type { ASTNode } from '../language/ast';
+
+/**
+ * Produces a string for formatting a syntax or validation error with an
+ * embedded location.
+ */
+export function formatError(
```

Comment:
We already have another function called `formatError` which does something slightly different.

How about `printLocatedError`?